src/components/molecules/AssetListTitle.tsx

```diff
@@ -19,8 +19,7 @@ export default function AssetListTitle({
   const [assetTitle, setAssetTitle] = useState<string>(title)
 
   useEffect(() => {
-    if (assetTitle || !config?.metadataCacheUri) return
-
+    if (title || !config?.metadataCacheUri) return
```

Comment:
the update issue was here, for some reason the table did not refresh the components and they kept their state, so `assetTitle` was always set with the values of the first page. Will test a bit more so see if this impacts the rest of the site

Reply:
seems to not break anything